cases/cosmo-ghg-11km-test/config.py

```diff
@@ -1,17 +1,26 @@
 import os
+"""
+Configuration file for the 'cosmo-ghg-11km-test' case with COSMO-GHG
+"""
 
 # GENERAL SETTINGS ===========================================================
 user = os.environ['USER']
+if user == 'jenkins':
+    compute_account = 'g110'
+elif os.path.exists(os.environ['HOME'] + '/.acct'):
+    with open(os.environ['HOME'] + '/.acct', 'r') as file:
+        compute_account = file.read().rstrip()
+else:
+    compute_account = os.popen("id -gn").read().splitlines()[0]
+compute_host = 'daint'
+compute_queue = 'debug'  # 'normal'
+constraint = 'gpu'  # 'mc'
+
 target = 'cosmo-ghg'
 restart_step = 12  # hours
 #subtarget = 'spinup'
 #spinup = 6
 
-compute_host = 'daint'
-compute_queue = 'debug'  # 'normal'
-compute_account = 'em05'
-constraint = 'gpu'  # 'mc'
-
 if constraint == 'gpu':
     ntasks_per_node = 12
     mpich_cuda = (
```

Comment:
This comment is actually about four lines below, but Github doesn't allow me to make a comment there. 
`'export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:/opt/cray/nvidia/default/lib64`
This has no effect because the added folder doesn't exist (anymore?) on Daint. Anyhow, it's also not needed (anymore?). So you can remove that line. It's also present in the test-case `cosmo-7` where you could remove it.

Reply:
Yes, there are probably some leftovers from past ages. You can try to make those changes and push directly to the `jenkins` branch.